recipe/meta.yaml

```diff
@@ -49,3 +48,4 @@ extra:
     - ocefpaf
     - JiaweiZhuang
     - raphaeldussin
+    - huard
```

Comment:
@huard you should be able to merge it next time ;-p

Reply:
Thanks!